### ISSUE TITLE
Docker: Need a start training script

### ISSUE THREAD

**Issue by LoicNyssen** [NES-NN/OpenAI-Testbed#11]
Need a script to perform pre training tasks and clean up tasks. 

for example, before starting training, set DISPLAY and close all emulators after training. 

Also might be nice to have stdout also logged to a file.

**Comment by jbeemster** [comment]
This can be done from wooey - do we need anything else for this ticket?

**Comment by jbeemster** [comment]
Closing as not needed.
